staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/marshal_cbor.go

```diff
@@ -0,0 +1,35 @@
+package v1
+
+import (
+	"github.com/fxamacker/cbor/v2"
```

Comment:
This should use the Marshal func from apimachinery. It's configured to behave in all the details like the Serializer implementation does.

```suggestion
	cbor "k8s.io/apimachinery/pkg/runtime/serializer/cbor/direct"
```